packages/client/src/giscus.tsx

```diff
@@ -15,16 +12,14 @@ export interface GiscusProps {
 
 let GiscusKeyInc = 0;
 
-export function Giscus({ category, categoryId, repo, repoId }: GiscusProps) {
+const Giscus: FC<GiscusProps> = ({ category, categoryId, repo, repoId }) => {
```

Comment:
```suggestion
const Giscus = ({ category, categoryId, repo, repoId }: GiscusProps) => {
```

Reply:
We can discuss of forbidding `FC`, `FunctionalComponent` imports from react in our future guild config ;)